.github/workflows/testing-scheduled.yml

```
@@ -5,16 +5,16 @@ on:
     - cron: '30 5 * * 1' 
 
 
-jobs:     
-  test-stable-scheduled:
-    uses: 
-      ./.github/workflows/testing.yml
-    with:
-      xdist: no
-      branch_name: stable
-      event_name: ${{ github.event_name }}
-    secrets:
-      PAT: ${{ secrets.PAT }}
+jobs:
+  trigger-testing-stable:
+    runs-on: ubuntu-latest
+    permissions:
+        actions: write
```

```suggestion
      actions: write
```

This should just be indented with two spaces more compared to the previous line, to stay consistent with the rest of the file.